projects/experimental/components/compass/compass.style.less

```diff
@@ -0,0 +1,37 @@
+@import 'taiga-ui-local';
+
+:host {
+    position: relative;
+    display: block;
+    width: 1rem;
```

Comment:
It's supposed to be 1rem + border of 3px

Reply:
fixed